src/patternfly/components/Breadcrumb/breadcrumb.scss

```diff
@@ -53,6 +54,10 @@
   white-space: nowrap; // Keeps the item-divider and __link text on the same line
   list-style: none;
 
+  &:first-child {
+    margin-inline-start: calc(var(--#{$breadcrumb}__link--PaddingInlineStart) * -1); // left align first item in list with list bounds while retaining link padding
+  }
```

Comment:
Currently, breadcrumbs are misaligned when link is present in the first child. 

<img width="762" alt="Screenshot 2024-02-03 at 12 10 18 PM" src="https://github.com/patternfly/patternfly/assets/5385435/c6c23a40-bb14-4728-a11c-feb4d2c629c0">

Removing padding from the first link resolves this issue:

<img width="808" alt="Screenshot 2024-02-03 at 12 11 48 PM" src="https://github.com/patternfly/patternfly/assets/5385435/ee24646b-06d4-4961-bec9-a0de04ccc990">

Reply:
Oh I was asking why the padding was added in the first place. It doesn't exist in v6 currently. This last example is misaligned but that's because the browser/agent styles are adding padding to the breadcrumb link when it's a button. IMO that's just a bug we overlooked since we don't want the default `button` element styles - we'd just remove that (set breadcrumb__link to `padding: 0`). 

<img width="1757" alt="Screenshot 2024-02-05 at 2 57 40 PM" src="https://github.com/patternfly/patternfly/assets/35148959/a1628c62-320b-4101-97cf-2a32931b56ca">

This is what I'm seeing in this PR currently

<img width="1757" alt="Screenshot 2024-02-05 at 3 01 21 PM" src="https://github.com/patternfly/patternfly/assets/35148959/fdd986b2-1c64-4b80-bf40-9a5c2584fae2">